readme.md

```diff
@@ -38,63 +38,63 @@ $ npm -v
 
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
-* **Your Answer:**
+* **Your Answer:** forking a git repository copies copies the repo, including all previous commits to another account and maintains the connection to the original repo, whereas cloning simply copies the repository from the current state of the repository.
 
 ---
 
 - [ ] Run `npm init -y` from the command line
 
 * **Question:** What does `npm init` do? How does the `-y` flag modify that command?
 
-* **Your Answer:**
+* **Your Answer:** `npm init` creates the package.json file for the project? Don't know what the -y flag does.
```

Comment:
Try running `npm init` in an empty folder. Then, delete the generated `package.json` and run `npm init -y`. That'll help clarify the difference!

Reply:
Ah, okay I tried it inside an empty project folder and saw the difference. Thank you.